group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/OptimizedUniformAssignmentBuilderTest.java

```diff
@@ -61,15 +63,12 @@ public class OptimizedUniformAssignmentBuilderTest {
 
     @Test
     public void testOneMemberNoTopicSubscription() {
+        MetadataImage metadataImage = new MetadataImageBuilder()
+            .addTopic(topic1Uuid, topic1Name, 3)
+            .addRacks()
```

Comment:
nit: Is `addRacks` necessary here and in all the other places?

Reply:
Currently, assignor doesn't use `racksForPartition`, so it's not necessary. Remove it from assignor test cases.